### PR TITLE
Show post`s create date in hint

### PR DESCRIPTION
**Pull request by tavriaforever**
// cc @gela-d

### CODE REVIEW

File: src/blocks/common.blocks/post/__date/post__date.bemhtml [hunk 1]
```diff
@@ -3,25 +3,50 @@ block('post').elem('date')(
...
+            date.setTime(postDate);
 
-            return d + ' ' + BEM.I18N('post', 'month' + m) + ' ' + y;
+            return [
+                date.getDate(),
+                ' ',
+                BEM.I18N('post', 'month' + date.getMonth()),
+                ' ',
+                date.getFullYear()
+            ].join('');
```
Comment (tadatuta) [inline src/blocks/common.blocks/post/__date/post__date.bemhtml]: you may get rid of `' '` items if use `join(' ')` (and the same bellow)

File: src/blocks/common.blocks/post/__date/post__date.bemhtml [hunk 2]
```diff
@@ -3,25 +3,50 @@ block('post').elem('date')(
...
+         * @returns {string} example: changed 14 match 2015
+         */
+        var getFormattedDate = function (postDate) {
             var date = new Date();
-            date.setTime(postDate);
 
-            var d = date.getDate(),
-                m = date.getMonth(),
-                y = date.getFullYear();
+            date.setTime(postDate);
```
Comment (tadatuta) [inline src/blocks/common.blocks/post/__date/post__date.bemhtml]: use `new Date(postDate)` instead

File: src/blocks/common.blocks/post/__date/post__date.bemhtml (hunk 1 above)
Comment (tavriaforever) [inline src/blocks/common.blocks/post/__date/post__date.bemhtml]: yes, you are right

File: src/blocks/common.blocks/post/__date/post__date.bemhtml (hunk 2 above)
Comment (tavriaforever) [inline src/blocks/common.blocks/post/__date/post__date.bemhtml]: ok

**Comment by tavriaforever** [comment]
Refactor code, update PR

**Comment by tadatuta** [comment]
lgtm

**Comment by gela-d** [comment]
ok

**Pull request merged**